src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService newService = new UserService();
```

Comment:
better make it a class-level variable ;)

Reply:
But in my code it is already made as a class-level variable:
public class AuthenticationService {
       public boolean login(String email, String password) {
        UserService newService = new UserService();
        User user = newService.findByEmail(email);
        return user != null && user.getPassword().equals(password);
    }
}
Can you explain your comment in more details, please?

Reply:
I've understood th issue, please, disregard

Reply:
it means, that you shouldn't create `UserService newService = new UserService();` in `public boolean login(String email, String password)`, but create it as class-level variable

```
public class AuthenticationService {

    private final UserService newService = new UserService();
```